packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1177,7 +1186,7 @@ enum FocusHighlightStrategy {
 ///    a given [BuildContext].
 ///  * The [focusManager] and [primaryFocus] global accessors, for convenient
 ///    access from anywhere to the current focus manager state.
-class FocusManager with DiagnosticableTreeMixin {
+class FocusManager with DiagnosticableTreeMixin, ChangeNotifier implements Diagnosticable {
```

Comment:
Can you explain why it's necessary to declare 'implements Diagnosticable' here? I suspect this is related to an issue we have in our own code.

Reply:
Yeah, the short answer is that it's just something I missed doing in a previous PR, and it's temporary.

There are some places (now only inside Google: I fixed all the ones in Flutter) that cast the results of calling `toDiagnosticsNode` to a `Diagnosticable`, and if the class only implements `DiagnosticableMixin`, then that cast throws. This is a workaround for that case. See https://github.com/flutter/flutter/pull/49726 for more detail.

Once `Diagnosticable` is removed in favor of `DiagnosticableMixin`, this will go away too.

Reply:
Ah, OK, that's a different issue than I thought then. (If you're curious ping me offline, it's not relevant to this change.)